examples/cloudobject.py

```diff
@@ -0,0 +1,27 @@
+"""
+Simple PyWren example using cloudobjects to transparently pass
+objects stored in the storage backend between functions without
+knowing they exact location (bucket, key)
+"""
+import pywren_ibm_cloud as pywren
+
+
+def my_function_put(text, internal_storage):
+    co1 = internal_storage.put_object('Temp object test 1: {}'.format(text, ))
+    co2 = internal_storage.put_object('Temp object test 2: {}'.format(text, ))
+    return [co1, co2]
+
+
+def my_function_get(co, internal_storage):
+    data = internal_storage.get_object(co)
```

Comment:
@JosepSampe the "co" is actually an array of [co1, co2], right? ( this is what returned from my_function_put? but then my_function_get does get on the array? and get a single  variable "data" ?

Reply:
@gilv Nop, "co" i only one CloudObject. In the example the first function returns an array of 2 CloudObjects [co1, co2]. This is the input of the 2nd function, This means that the map() spawns 2 functions, each one process one CloudObject.

Reply:
@JosepSampe right, my fault...